bangazonapi/views/paymenttype.py

```diff
@@ -80,11 +80,14 @@ def destroy(self, request, pk=None):
     def list(self, request):
         """Handle GET requests to payment type resource"""
         payment_types = Payment.objects.all()
-
-        customer_id = self.request.query_params.get('customer', None)
+        
+        try:
+            customer_id = Customer.objects.get(user=request.auth.user)
```

Comment:
Here you refer to `customer_id`. Is it really the id or is it the whole `customer` object?